src/ontology/cl-edit.owl

```diff
@@ -27772,26 +27780,6 @@ AnnotationAssertion(rdfs:label obo:D96882F1-8709-49AB-BCA9-772A67EA6C33 "alobate
 EquivalentClasses(obo:D96882F1-8709-49AB-BCA9-772A67EA6C33 ObjectIntersectionOf(obo:GO_0005634 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0002506)))
 SubClassOf(obo:D96882F1-8709-49AB-BCA9-772A67EA6C33 obo:GO_0005634)
 
-# Class: obo:PR_000013502 (parvalbumin alpha)
```

Comment:
Not quite sure what happened here, but basically there was extra labels to their synonym names, so I cleared it all so it would revert back to PR import names.